src/main/scala/com/fulcrumgenomics/umi/ExtractUmisFromBam.scala

```diff
@@ -76,7 +76,8 @@ class ExtractUmisFromBam
   @arg(flag = "r", doc = "The read structure, one per read in a template.")      val readStructure: Seq[String],
   @arg(flag = "b", doc = "SAM tags in which to store the molecular barcodes (one-per segment).")
                                                                                  val molecularBarcodeTags: Seq[String] = Seq.empty,
-  @arg(flag = "a", doc = "Annotate the read names with the molecular barcodes. See usage for more details.") val annotateReadNames: Boolean = false
+  @arg(flag = "a", doc = "Annotate the read names with the molecular barcodes. See usage for more details.") val annotateReadNames: Boolean = false,
+  @arg(flag = "x", doc = "The SAM tag with clipping information.") val clippingAttribute: Option[String] = None
```

Comment:
A few quick things:
1. `-x` seems like an odd choice for flag.  How about either `-c` for clipping, or just omit the flag altogether?
2. A little more documentation might be nice, e.g. "The SAM tag with the position in read to clip adapters (e.g. XT as produced by Picard's MarkIlluminaAdapters".
3. Are there ever other tags anyone would want to do the same with such that this should take a `Seq` and adjust multiple?